README.md

```diff
@@ -223,8 +223,10 @@ Some custom one shot tasks are available.
 
 To run a custom task: `mix <custom task>`
 
-* `Transport.ImportAom`: import the aom data from the cerema
+* `Transport.ImportAOMs`: import the aom data from the cerema
+* `Transport.Import_communes`: import the french communes from data.gouv
 * `Transport.ImportEPCI`: import the french EPCI from data.gouv
+* `Transport.Import_departements`: import the french Départements from data.gouv
 * `Transport.OpenApiSpec`: generate an OpenAPI specification file
```

Comment:
Merci d'avoir mis à jour ceci ! Dommage qu'on [ne puisse pas lister les tasks](https://elixirforum.com/t/get-a-list-of-all-mix-tasks-in-lib-mix-tasks/58844).

Je pense qu'il y a une confusion entre nom de la classe dans la code et la commande à lancer. `Transport.ImportAOMs` vs `mix transport.import_aoms`. Ou alors les deux fonctionnent ? Dans ce cas il faudrait

```diff
- Transport.Import_departements
+ Transport.ImportDepartements
```

Reply:
Alors, en vrai, la commande dérive du nom de la classe, avec une transformation PascalCase > snake_case, voir https://elixirforum.com/t/how-to-work-with-tasks-named-after-acronyms/13647/4 Mais les deux vont marcher, tu peux faire `mix Transport.ImportDepartements` ou` mix `transport.import_departements`

D’où ma transformation initiale du nom des classes :
- nom de module `ImportEPCI` = tâche `mix transport.import_e_p_c_i` (ce qui est moche) ou `mix Transport.ImportEPCI`
- nom de module `ImportEpci` = tâche `mix transport.import_epci` ou `mix transport.ImportEpci`

Je vais homogénéiser en gardant les acronymes en majuscules et en suggérant la tâche en PascalCase dans le Readme.